tests/commands/entry_break.py

```diff
@@ -13,9 +13,17 @@ def test_cmd_entry_break(self):
         gdb = self._gdb
 
         # run once (ok)
-        res = gdb.execute("entry-break", to_string=True).strip()
-        assert res.startswith("[+] Breaking at")
+        lines = (gdb.execute("entry-break", to_string=True) or "").strip().splitlines()
+
+        #
+        #
+        # "[+] Breaking at Breaking at entry-point" might be the 1st or 2nd line (depending on whether we target
+        # a PIC binary) start with
+        #
+        assert len(lines) >= 2
+        assert lines[0].startswith("[+] Breaking at entry-point") or \
```

Comment:
why not just do something like `assert any(line.startswith("[+] Breaking at entry-point") for line in lines)`

Reply:
Fixed